basilisk/tests/test_edit_block_dialog.py

```
@@ -0,0 +1,69 @@
+"""Comprehensive unit tests for EditBlockDialog."""
+
+import pytest
+from unittest.mock import Mock, MagicMock, patch, PropertyMock
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)